moxin-protocol/src/data.rs

```diff
@@ -40,17 +44,17 @@ pub struct DownloadedFile {
 // We're using the HuggingFace identifier as the model ID for now
 // We should consider using a different identifier in the future if more
 // models sources are added.
-#[derive(Debug, Clone, Default)]
+#[derive(Debug, Clone, Default, serde::Serialize, serde::Deserialize)]
 pub struct Model {
     pub id: ModelID,
     pub name: String,
     pub summary: String,
     pub size: String,
     pub requires: String,
     pub architecture: String,
-    pub released_at: NaiveDate,
+    pub released_at: DateTime<Utc>,
```

Comment:
The time data obtained from Hugging Face is in UTC, and I think it should be changed to UTC here. Then, when using it, convert it to NativeDate or NativeDateTime for better compatibility.

Reply:
Are you okay with the modifications made to the released_at?

@jmbejar

Reply:
Makes totally sense 👍